src/main/java/org/myrobotlab/service/interfaces/Translator.java

```diff
@@ -1,8 +1,29 @@
 package org.myrobotlab.service.interfaces;
 
 public interface Translator {
-  
-    public String translate(String text);
-    
-    public String translate(String text, String fromLang, String toLang);
+
+  /**
+   * Translate the incoming text from incoming language to target language This
+   * method will rely on the internal state of the translation service to
+   * determine incoming and target language
+   * 
+   * @param text
+   * @return
+   */
+  public String
+
+      translate(String text);
```

Comment:
This interface should allow you to set the source/ destination language .. o/w the interface won't support changing which language you are translating to by default.

Reply:
but thats really setting the bot

Reply:
removed